src/toil/test/utils/utilsTest.py

```diff
@@ -115,6 +115,25 @@ def statusCommand(self, failIfNotComplete=False):
             commandTokens.append('--failIfNotComplete')
         return commandTokens
 
+    def test_config_functionality(self):
+        """Ensure that creating and reading back the config file works"""
+        config_file = os.path.abspath("config.yaml")
+        config_command = [self.toilMain, 'config', config_file]
+        try:
+            subprocess.check_call(config_command)
+        except subprocess.CalledProcessError:  # This happens when the script fails due to having unfinished jobs
+            self.assertRaises(subprocess.CalledProcessError, system)
```

Comment:
This looks like it invokes the `system` function with no arguments and makes sure it raises an error, but I can't figure out what it is doing here.